Makefile

```diff
@@ -32,9 +33,8 @@ vet:
 	@$(GO) vet $(pkgs)
 
 build:
-	@echo ">> building binaries"
-	@./build/assets.sh
-	@./build/build.sh
+	@echo ">> building binaries using docker"
+	@docker run -i -v $(PWD):/go/src/github.com/google/cadvisor golang:$(goversion) /bin/sh -c "cd /go/src/github.com/google/cadvisor && go get github.com/tools/godep && ./build/assets.sh && ./build/build.sh"
```

Comment:
@timstclair 

#1253 will still be in effect. Adding a Dockerfile makes sense. I'm updating the PR now.